src/datascraping/lunar_data.py

```
@@ -0,0 +1,102 @@
+            time = i[1]
+            if 'am' in time:
+                time = time.split()[0]
+                if len(time) <= 4:
+                    time = f'0{time}'
+            elif 'pm' in time:
+                nums = time.split()[0]
+                hrs, mins = nums.split(':')
+                if int(hrs) > 12:
+                    hrs = int(hrs) + 12
+                else:
+                    hrs = int(hrs)
```

Ruff (0.12.2)

Use ternary operator `hrs = int(hrs) + 12 if int(hrs) > 12 else int(hrs)` instead of `if`-`else`-block

Replace `if`-`else`-block with `hrs = int(hrs) + 12 if int(hrs) > 12 else int(hrs)`

(SIM108)